src/Components/Filter/Filter.css

```diff
@@ -0,0 +1,64 @@
+.filter-section-wrapper {
+    border: 1px solid var(--clr-grey-light);
+    border-radius: var(--border-radius);
+    padding: var(--padding-md);
+    box-shadow: var(--box-shadow-light);
+    background-color: var(--clr-whitesmoke);
+    width: max-content;
+    position: absolute;
+    top: 4rem;
+    left: -5.5rem;
+    z-index: 5;
```

Comment:
Utility for z-index?

Reply:
Will be resolving in the next PR, thanks for pointing it